snowplow-demo-compose/proguard-rules.pro

```diff
@@ -18,4 +18,19 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+# Fixes R8 full mode crash
```

Comment:
We should also add these to the [setup section in the docs](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/installation-and-set-up/android-tracker/).